broker/thinBroker.go

```diff
@@ -1848,7 +1853,7 @@ func (tb *ThinBroker) RegisterCSource(w rest.ResponseWriter, r *rest.Request) {
 				context = append(context, linkMap["rel"]) // Make use of "link" and "type" also
```

Comment:
[Major] new attribute will not be added logic incorrect.

Reply:
Added support for same in code